week1/utils.py

```
@@ -0,0 +1,24 @@
+    plt.show()
+
+
+def plot_feature_importance(feature_importance, feature_names, top_n=10):
+    plt.figure(figsize=(10, 6))
+    sorted_idx = np.argsort(feature_importance)
+    pos = np.arange(sorted_idx[-top_n:].shape[0]) + .5
+    plt.barh(pos, feature_importance[sorted_idx[-top_n:]])
+    plt.yticks(pos, feature_names[sorted_idx[-top_n:]])
+    plt.title(f'Top {top_n} Feature Importance')
+    plt.tight_layout()
+    plt.show()
```

Ruff

No newline at end of file

Add trailing newline

(W292)